Doc/whatsnew/3.9.rst

```diff
@@ -195,6 +195,11 @@ which have not started running, instead of waiting for them to complete before
 shutting down the executor.
 (Contributed by Kyle Stanley in :issue:`39349`.)
 
+Removed daemon threads from :class:`~concurrent.futures.ThreadPoolExecutor`
+and :class:`~concurrent.futures.ProcessPoolExecutor`. This improves
```

Comment:
Using a tilde `~` shortens the Sphinx markup link to appear as just the last part, I.E. `ThreadPoolExecutor` and `ProcessPoolExecutor`. I typically use this when the classes are easily distinguishable based on their names alone, and the section already makes it clear what module it's in. This helps to make it a bit more succinct.